### PR TITLE
Sparcl nb and readme in tutorials/database/ folder for dr1 branch

### PR DESCRIPTION
**Pull request by stephjuneau**
This PR is for a new notebook for using SPARCL at NERSC to access DESI DR1 spectra and a brief README.md file in that same folder to explain what the notebooks are now there are two different ones.

### CODE REVIEW

**Comment by weaverba137** [comment]
Thank you for adding the `README`. I was thinking that might be helpful. I want to test the notebook again before merging. I probably won't have time for that today.

**Comment by stephjuneau** [comment]
OK, now I'm happy with this version. Cleaned up a bit the plots by using the mask. Ben is busy this morning so I'd like to know if someone else might like to take this new notebook for a spin? @forero or @sbailey or @moustakas ? No worries if y'all are too busy, I'll also post a version in the NERSC filesystem before the ECS tutorial.

**Review by weaverba137 — APPROVED** [review]
I ran the notebook at NERSC. Everything was fine. I have one modification to suggest for the `pip` command, but otherwise this can be merged.

File: database/How_to_use_SPARCL_at_NERSC.ipynb [hunk 1]
```diff
@@ -0,0 +1,992 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4cb3b11f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "## Uncomment the following only if SPARCL client is not already installed\n",
+    "#!pip install --upgrade sparclclient"
```
Comment (weaverba137) [inline database/How_to_use_SPARCL_at_NERSC.ipynb]: Add `--user` to the `pip` command.

Comment (stephjuneau) [reply database/How_to_use_SPARCL_at_NERSC.ipynb]: OK, just added and committed the change

**Comment by stephjuneau** [comment]
Hi @weaverba137 -- just made the change to add `--user` to the pip install command.

**Comment by weaverba137** [comment]
Merge when ready.

**Pull request merged**